core/src/main/java/org/apache/calcite/rel/rel2sql/SqlImplementor.java

```diff
@@ -890,6 +890,7 @@ private SqlNode callToSql(@Nullable RexProgram program, RexCall rex, boolean not
         }
       case PLUS:
       case MINUS:
+      case OTHER_FUNCTION:
```

Comment:
This doesn't look right. **OTHER_FUNCTION** case should not be clubbed with any other function, as there can be more than one functions mapped to **OTHER_FUNCTION**.

Reply:
OTHER_FUNTION is no longer clubbed with PLUS and MINUS SqlKinds and getOperatorForOtherFunc is created and used instead.